fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/_common/rest/FoDUrls.java

```diff
@@ -41,12 +41,15 @@ public class FoDUrls {
     public static final String RELEASE_SCANS = RELEASE + "/scans";
     public static final String STATIC_SCANS = ApiBase + "/releases/{relId}/static-scans";
     public static final String STATIC_SCANS_IMPORT = STATIC_SCANS + "/import-scan";
-    public static final String STATIC_SCAN_START = STATIC_SCANS + "/start-scan-advanced";
+    public static final String STATIC_SCAN_START = STATIC_SCANS + "/start-scan";
+    public static final String STATIC_SCAN_START_WITH_DEFAULTS = STATIC_SCANS + "/start-scan-with-defaults";
+    public static final String STATIC_SCAN_START_ADVANCED = STATIC_SCANS + "/start-scan-advanced";
     public static final String DYNAMIC_SCANS = ApiBase + "/releases/{relId}/dynamic-scans";
     public static final String DYNAMIC_SCANS_IMPORT = DYNAMIC_SCANS + "/import-scan";
     public static final String MOBILE_SCANS = ApiBase + "/releases/{relId}/mobile-scans";
     public static final String MOBILE_SCANS_IMPORT = MOBILE_SCANS + "/import-scan";
     public static final String MOBILE_SCANS_SETUP = MOBILE_SCANS + "/scan-setup";
     public static final String MOBILE_SCANS_START = MOBILE_SCANS + "/start-scan";
+    public static final String ENTITLEMENTS = ApiBase + "/tenant-entitlements";
```

Comment:
Yes, you are correct a "regular" user, for example "Developer" role wouldn't be able to access this endpoint. I will keep the commands but remove the use of entitlement validation - which is a shame since this is the only API call we can see whether it is still valid/not expired.

Reply:
Do we need access to this endpoint to work around some of the existing FoD API limitations/inconsistencies, or would it in general be useful to have the ability for regular users to query available entitlements? Or, conceptually speaking, do we want to have the ability to list/select/otherwise process available entitlements on the client side (in fcli or other integrations), or should this all ideally be handled on the FoD side (like automatically selecting entitlements based on preferences when configuring a scan)?

Can you comment on the ongoing email thread with the FoD team on what API functionality would be required/useful to have for use in fcli (and for what reason), and what user roles should be able to invoke these endpoints?